sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/jdbc/JDBCTable.scala

```diff
@@ -34,4 +38,16 @@ case class JDBCTable(ident: Identifier, schema: StructType, jdbcOptions: JDBCOpt
     capabilities.add(BATCH_READ)
```

Comment:
How about write?

Reply:
Will add `BATCH_WRITE`. Are these all we need to do to support `BATCH_READ` and `BATCH_WRITE`? I know you are working on `BATCH_READ`, but I thought there are more to do than just adding `BATCH_READ` capabilities. Sorry if you already have coded all these. @MaxGekk